src/lpc.jl

```diff
@@ -59,7 +66,7 @@ function implements the mathematics described in [1].
 """
 function lpc{T <: Number}(x::AbstractVector{T}, p::Int, ::LPCLevinson)
     R_xx = xcorr(x,x)[length(x):end]
-    a = zeros(p,p)
+    a = zeros(T,p,p)
```

Comment:
I've tested the versions for type stability - the Burg versions works with Float(16/32/64)/BigFloat - where as the Levinson version works only with Float(32/64) - because of conv() doing an FFT (which is alright for now I guess).
Anyhow - the Levinson version can also have the reflection coeffs returns eventually.
Test is cool, I'll try to add a Ladder(Lattice) filter - where the reflection coeffs are used, and that can also be added to the test